### ISSUE TITLE
Imputer was deprecated in version 0.20 and will be removed in 0.22. Import impute.SimpleImputer from sklearn instead.

### ISSUE THREAD

**Issue by ILoveMyWife** [Avik-Jain/100-Days-Of-ML-Code#62]
Describe the bug
**Imputer was deprecated in version 0.20 and will be removed in 0.22. Import impute.SimpleImputer from sklearn instead.**

To Reproduce

Day1
Step 3: Handling the missing data
**Imputer was deprecated in version 0.20 and will be removed in 0.22. Import impute.SimpleImputer from sklearn instead.**

OS: win10
Browser chrome
Additional context

I use 
**imputer = SimpleImputer(missing_values = "NaN", strategy = "mean")**
to replace this 
**imputer = Imputer(missing_values = "NaN", strategy = "median", axis = 0)**
but get an error:
**Input contains NaN, infinity or a value too large for dtype('float64').**

**Can anybody help me about this ,THANKS**

**Comment by lflucasferreira** [comment]
I changed the code for this:

> from sklearn.impute import SimpleImputer
> imputer = SimpleImputer(strategy="mean")
> imputer = imputer.fit(X[:, 1:3])
> X[:, 1:3] = imputer.transform(X[:, 1:3])

As well after you will need to change for this:
> from sklearn.model_selection import train_test_split
> X_train, X_test, Y_train, Y_test = train_test_split( X , Y , test_size = 0.2, random_state = 0)

The last change is:
> from sklearn.preprocessing import StandardScaler
> sc_X = StandardScaler()
> X_train = sc_X.fit_transform(X_train).astype(float)
> X_test = sc_X.fit_transform(X_test).astype(float)

**Comment by ILoveMyWife** [comment]
> I changed the code for this:
> 
> > from sklearn.impute import SimpleImputer
> > imputer = SimpleImputer(strategy="mean")
> > imputer = imputer.fit(X[:, 1:3])
> > X[:, 1:3] = imputer.transform(X[:, 1:3])
> 
> As well after you will need to change for this:
> 
> > from sklearn.model_selection import train_test_split
> > X_train, X_test, Y_train, Y_test = train_test_split( X , Y , test_size = 0.2, random_state = 0)
> 
> The last change is:
> 
> > from sklearn.preprocessing import StandardScaler
> > sc_X = StandardScaler()
> > X_train = sc_X.fit_transform(X_train).astype(float)
> > X_test = sc_X.fit_transform(X_test).astype(float)

Thanks a lot
